main.tf

```diff
@@ -24,6 +25,23 @@ resource "aws_cloudfront_origin_access_control" "this" {
   signing_protocol                  = each.value["signing_protocol"]
 }
 
+resource "aws_cloudfront_vpc_origin" "this" {
```

Comment:
Missing `tags` in this resource

Reply:
Nice catch. I will add it in my next commit.